presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetCompressionUtils.java

```diff
@@ -45,23 +47,22 @@ public static Slice decompress(CompressionCodecName codec, Slice input, int unco
             case GZIP:
                 return decompressGzip(input, uncompressedSize);
             case SNAPPY:
-                return decompressSnappy(input, uncompressedSize);
+                return decompress(new SnappyDecompressor(), input, uncompressedSize);
+            case LZO:
+                return decompress(new LzoDecompressor(), input, uncompressedSize);
```

Comment:
Hmm I thought it was enabled. When I enabled that lzo, tests failed. The weird thing is when I plug in the hadoop lzo decompressor and run the tests they all pass. So this may be an issue with the airlift lzo implementation.

@dain any thoughts? To reproduce this you can use my branch [here](https://github.com/prestodb/presto/compare/master...nezihyigitbasi:airlift-lzo-issue) . This code currently enables the hadoop lzo decompressor in `ParquetCompressionUtils` and tests succeed. If you enable the airlift lzo decompressor it will fail.